.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'npm' # See documentation for possible values
+    directory: '/' # Location of package manifests
+    schedule:
+      interval: 'weekly'
+    open-pull-requests-limit: 50
+    # Disable rebasing for pull requests, as having several open pull requests all get simultaneously rebased gets
+    # noisy from a notification standpoint
+    rebase-strategy: 'disabled'
```

Comment:
@jaredcbaum this will lead to opening a new PR for any dependency that is not up to date in the project's root level `package.json`. Are there any (e.g. React) that we should _not_ update/do manually on a given cadence? If so, I can add an `ignore` list similar to https://github.com/ionic-team/stencil/blob/dd450aa6f2d2b208f01f195c7cdb1231b7ff635e/.github/dependabot.yml#L15 here. Just LMK what packages/versions we shouldn't update

Reply:
Nothing comes to mind after looking through the deps.

Reply:
Even if so, I'm sure we can fine tune after the fact.

Reply:
👍 Is there a max version of React I should put in here? e.g. No higher than 17.Y.Z? 17.0.Z?

Reply:
https://github.com/ionic-team/stencil-site/blob/main/ionic-preset-classic/package.json#L37